gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/JvmApplicationBuildTypes.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.desktop.application.dsl
+
+import org.gradle.api.Action
+import org.gradle.api.model.ObjectFactory
+import org.jetbrains.compose.desktop.application.internal.new
+import javax.inject.Inject
+
+abstract class JvmApplicationBuildTypes @Inject constructor(
+    objects: ObjectFactory
+) {
+    internal val default: JvmApplicationBuildType = objects.new("")
```

Comment:
Why empty String?

Reply:
The default build type.
It is similar concept to android debug/release build profiles.
A build profile's name is added to task's name (the pattern is `<taskAction><buildTypeName><taskSubject>`).
I don't want to break compatibility with the existing plugin by effectively renaming e.g. `packageDmg` to `packageDebugDmg`. Treating debug/dev profile as default profile without name makes sense to me.
What do you think, @dima-avdeev-jb ?

Reply:
Yeah, good decision to save backward compatibility!